helpers/UrgentEmail.js

```diff
@@ -0,0 +1,50 @@
+import Postals from './Postals';
+
+/**
+ *
+ *
+ * @export
+ * @class urgentEmail
+ */
+export default class UrgentEmail {
+  /**
+  *
+  * @static
+  * @param {string} recipient - recipient of the email
+  * @param {string} name - name of the recipient
+  * @param {object} payloadDetails - payload details
+  * @memberof EmailNotification
+  * @returns {function} - returns a function call
+  */
+  static sendUrgentEmail(recipient, name, payloadDetails) {
```

Comment:
Hi Victor, could this method name be changed to a more one? This one seems a little too vague

Reply:
I get your point, it has been changed. Thank you.